### PR TITLE
n: update to 9.2.1

### PR DESCRIPTION
**Pull request by akierig**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

File: srcpkgs/n/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'n'
 pkgname=n
-version=9.2.0
+version=9.2.1
 revision=1
 build_style=gnu-makefile
 depends="curl"
```
Comment (sgn) [inline srcpkgs/n/template:6]: This is a simple `bash` script, that uses `curl` or `wget` to fetch `glibc`-only nodejs binaries distribution, then uses `tar` and `gzip` or `xz` to unpack it out.

So:
- **Personally**, I will not merge this update
- Its dependency needs to be updated
- It should be limited to glibc only.

Comment (akierig) [reply srcpkgs/n/template]: I think the most recent revision should cover everything

**Comment by akierig** [comment]
thanks @classabbyamp, should all be resolved.

File: srcpkgs/n/files/README.voidlinux [hunk 2]
```diff
@@ -1,3 +1,7 @@
 It is strongly recommended to set the environment variable N_PREFIX to install to a user-writable area. For example:
 
-export N_PREFIX=$HOME/local/n
+N_PREFIX=$HOME/local/n
+
+By default, nodejs only provides official builds for glibc. For musl and other unofficial supported builds, set the variable like so:
```
Comment (classabbyamp) [inline srcpkgs/n/files/README.voidlinux:5]:
```suggestion
By default, nodejs only provides official builds for glibc. For musl and other unofficially supported builds, set the variable like so:
```

**Pull request merged**